src/contexts/GUIContextProvider.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import { createMuiTheme, ThemeProvider } from '@material-ui/core';
+import React from 'react';
+import useLocalStorage from '../hooks/useLocalStorage';
+import { light } from '../components';
+
+interface DrawerContextProps {
+  drawer: string;
+  setDrawer: React.Dispatch<React.SetStateAction<string>>;
+}
+interface GUIContextProviderProps {
+  children: React.ReactElement;
+}
+
+export const DrawerContext = React.createContext({} as DrawerContextProps);
```

Comment:
If you add this simple function to the context

```
export function useGUIContext() {
  return useContext(DrawerContext);
}
```

Then on your children components, you just can access to the context  like this:

```
const {drawer} = useGUIContext();
```

The benefit is that imports and usage becomes easier every time you need to use this.